gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "12.10.10",
+  "version": "12.10.11",
```

Comment:
No version bumps on these tickets.  The version was bumped once on the feature branch and the purpose of the feature branch is to avoid having to do more version management in each of these tickets (and avoid wasting a bunch of version numbers)

Reply:
> No version bumps on these tickets. The version was bumped once on the feature branch and the purpose of the feature branch is to avoid having to do more version management in each of these tickets (and avoid wasting a bunch of version numbers)

I reverted the version bumps here: [ec11651](https://github.com/sonatype/sonatype-react-shared-components/pull/925/commits/ec11651d843e04bd040a7a5dde04343891a6f59b)